Source/JavaScriptCore/assembler/AssemblerBuffer.h

```diff
@@ -344,7 +344,7 @@ namespace JSC {
         void putInt64(int64_t value) { putIntegral(value); }
 #endif
         void putIntUnchecked(int32_t value) { putIntegralUnchecked(value); }
-        void putInt(int32_t value) { putIntegral(value); }
+        ALWAYS_INLINE void putInt(int32_t value) { putIntegral(value); }
```

Comment:
Shouldn't we inline `putIntUnchecked`, `putIntegral`, `putIntegralUnchecked`, and their variants while we're at it?